### ISSUE TITLE
Windows version

### ISSUE THREAD

**Issue by anewuser** [lutris/lutris#581]
There isn't anybody working on a Windows version yet, but please leave this open for people to get notified when/if somebody takes it.

**NOTICE**: if you just want to be notified, click on the "Subscribe" button on the Github sidebar instead of leaving comments that generate useless notifications.

**Comment by strycore** [comment]
A few people have mentioned being interested in a Windows (or Mac) version but no one has step up to do the actual port yet ^^

**Comment by TingPing** [comment]
A lot of the work I've done like dropping pyxdg, pyinotify, dbus-python, etc makes it a lot more portable to Win32/OSX.

So I would imagine the bulk of the work is in the runners, things like discovering installed games won't work, downloading linux binaries ofc won't work, etc.

**Comment by strycore** [comment]
Yes, the whole binary compatibility is just a matter of providing another architecture, we have plans to support ARMv7 already. The actual tricky part are all the linux specific bits in the client like the process monitor using files in `/proc` or commands like `fuser`.

**Comment by necrose99** [comment]
import freeze or py2exe or pyinstaller ,  might be able to get a windows binary if you have all the python deps. as it will make DLL's etc.  pyinstaller seen used for [openport-ssh <cloud revers ssh>](https://bitbucket.org/jan_de_bleser/openport-client) on linux to make thier Macos or windows/Linux static binaries.  anyhow for windows tinkers , python setup.py bdist_wheel , egg etc some bdist_py2exe /freeze /pyinstaller , etc may/may not work might req some fiddling.

**Comment by aaronfranke** [comment]
Glad to see this is a milestone for 1.0. I cannot recommend this to any of my friends if it does not run on Windows. It seems essential for Lutris to support Windows, as the entire point is to unify multiple "runners" and store platforms like Steam. A Mac version would be really nice for unifying things too.

**Comment by strycore** [comment]
I put 1.0 as the milestone because I wanted to tag all issues with a milestone and 1.0 was the furthest one away. That doesn't change the fact that no one seems interested in developing this port. It's not so much a "when will we do it" and more a "who will do it" problem.

**Comment by TingPing** [comment]
I forget the name of it but there is a Windows only application with a similar goal that tbh might just be the project that makes more sense on that platform. I *personally* believe Lutris should double down on linux with containerization and whatnot.

**Comment by strycore** [comment]
@TingPing are you refering to Horizon Launcher? That project is pretty much dead right now. 
I also personally believe that the current team should focus exclusively on Linux, not that a Windows version would be out the question but I have serious doubt that someone will ever make one.

**Comment by TingPing** [comment]
No it was something in C#.

**Comment by aaronfranke** [comment]
https://playnite.link/ - but really, having two programs for each OS is disunified. I have also requested Linux support for them, and will personally use whatever supports both.

**Comment by strycore** [comment]
Oh, that's kinda cool, I bet we can use their project to get better support for Origin, Uplay and Battle.net

**Comment by strycore** [comment]
I also wanted to point out that one of the goals I had for the project was to make people want to switch to Linux. That's doesn't mean I'm against a Windows version but I won't help making it happen.

**Comment by trymeouteh** [comment]
I am no expert with gaming on Linux but Windows is the king of gaming operating systems, A new OS called ReactOS look promising which is Windows but an open source version that is privacy friendly. I hope ReactOS takes off and if it does I may not switch to Linux for gaming and use ReactOS instead and if that is the case it would be nice to have Lutris on Windows so I can install the Windows version on ReactOS.

**Comment by strycore** [comment]
@trymeouteh let me know when you've managed to run a video game on ReactOS. I'll be right here, playing video games such as The Witcher 3, Quake Champions and Warframe on Linux.
